### DISCUSSION TITLE
Running multiple backups at the same time?

### DISCUSSION THREAD

**Discussion by mike-cuche** [borgbase/vorta#1905] [FAQ]
Hello.
Prior to vorta 0.9 I was able to run automatically multiple backups at once, when I plugged a specific external hard disk. I have a "local" profile that backups on said disk and a "remote" one that backups on a cloud.
I achieved that by using a couple of systemd user units:

```
[Unit]
Description=Vorta daemon

[Service]
Type=forking
ExecStart=/usr/bin/vorta -d

[Install]
WantedBy=default.target
```

```
[Unit]
Description=Run Vorta when I plug my external HDD
Requires=vorta-daemon.service
After=vorta-daemon.service

[Service]
Type=oneshot
ExecStart=/usr/bin/vorta
ExecStart=/usr/bin/vorta --create locale
ExecStart=/usr/bin/vorta --create remote


[Install]
WantedBy=run-media-user-BackupDisk.mount
```

However, when I upgraded to Vorta 0.9, now only one backup is able to start and now I'm forced to constantly check the application to verify that the first backup has ended and manually start the second one.
Creating the second backup from the terminal reports
`2024-01-21 08:42:13,647 - vorta.application - INFO - Creating backup using existing Vorta instance.`
but nothing happens.

This new behavior is annoying.
Is there a way to run multiple backups with this new version?

**Comment by m3nu** [comment]
Yes, this was already raised elsewhere. Making this an issue to track it.
